test/prototype/test_low_bit_optim.py

```diff
@@ -150,6 +150,57 @@ def test_optim_smoke(self, optim_name, dtype, device):
 
         for p1, p2 in zip(model.parameters(), model2.parameters()):
             torch.testing.assert_close(p2, p1)
+    
+    @pytest.mark.skipif(
+        not TORCH_VERSION_AT_LEAST_2_4, reason="FP8 CUDA requires PyTorch >= 2.4"
+    )
+    @pytest.mark.skipif(
+        torch.cuda.get_device_capability() < (8, 9), reason="FP8 CUDA requires compute capability >= 8.9"
+    )
```

Comment:
Running this will fail on CPU CI

Reply:
Since float8 can only be run on hopper GPU's i added test. should i remove it?

Reply:
The check should be inside the function, not as a decorator, like how it is done in `test_optim_smoke()`. Not exactly ideal, but we have to do it that way, since you can't call `torch.cuda.*` with CPU PyTorch.

Reply:
understood i will work on it